jwst/ramp_fitting/tests/test_ramp_fit.py

```diff
@@ -7,9 +7,17 @@
 from jwst.datamodels import dqflags
 from jwst.datamodels import RampModel
 
-DO_NOT_USE = dqflags.group['DO_NOT_USE']
-JUMP_DET = dqflags.group['JUMP_DET']
-SATURATED = dqflags.group['SATURATED']
+test_dq_flags = {
+    "DO_NOT_USE": dqflags.pixel["DO_NOT_USE"],
+    "JUMP_DET": dqflags.pixel["JUMP_DET"],
+    "SATURATED": dqflags.pixel["SATURATED"],
+    "NO_GAIN_VALUE": dqflags.pixel["NO_GAIN_VALUE"],
+    "UNRELIABLE_SLOPE": dqflags.pixel["UNRELIABLE_SLOPE"],
+}
```

Comment:
Again, no need for this.  Just use the existing dicts in `jwst.datamodels.dqflags.pixel` and `jwst.datamodels.dqflags.group`.

Reply:
I changed the code to `test_dq_flags = dqflags.pixel`.